mcp_server/server.py

```
@@ -0,0 +1,1794 @@
+# 远程 MCP 服务器配置（将在 load_servers_config 中初始化）
+# 支持两种格式：
+# - HTTP 服务器: "https://example.com/mcp" 或 {"type": "http", "url": "https://..."}
+# - stdio 服务器: {"type": "stdio", "command": "npx", "args": ["bing-cn-mcp"]}
+REMOTE_SERVERS: List[Union[str, Dict[str, Any]]] = []
+
+
+def load_servers_config():
+    """从文件加载服务器配置"""
+    global REMOTE_SERVERS
+    
+    # 首先从环境变量加载（优先级最高）
```

Ruff (0.14.8)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)